plugins/utilities/mood_light.py

```diff
@@ -0,0 +1,253 @@
+            parent=self._root_widget,
+            size=(200, 100),
+            position=(400, 150),
+            h_align="center",
+            v_align="center",
+            text="Limit brightness")
+
+        self.warn_text = ba.textwidget(
+            parent=self._root_widget,
+            text="",
+            size=(400, 200),
+            position=(150, 300),
+            h_align="center",
+            v_align="center",
+            maxwidth=600)
+
+        self.close_button = ba.buttonwidget(
+            parent=self._root_widget,
+            position=(550, 590),
+            size=(35, 35),
+            icon=ba.gettexture("crossOut"),
+            icon_color=(1, 0.2, 0.2),
+            scale=2,
+            color=(1, 0.2, 0.2),
+            extra_touch_border_scale=5,
+            on_activate_call=self.close,
+            button_type="square")
+
+        save_button = ba.buttonwidget(
+            parent=self._root_widget,
+            position=(520, 470),
+            size=(90, 70),
+            scale=1.5,
+            label="SAVE",
+            button_type="square",
+            on_activate_call=self.save_settings)
+
+        ba.textwidget(edit=self.upper_text, on_activate_call=ba.Call(self.on_text_click, "upper"))
+        ba.textwidget(edit=self.lower_text, on_activate_call=ba.Call(self.on_text_click, "lower"))
+
+    def save_settings(self):
+        with open("moodlightSettings.txt", "w") as mltxt:
+            data = "\n".join([str(Ldefault), str(Udefault)])
+            mltxt.write(data)
+        Print("settings saved")
+        self.close()
+
+    def close(self):
+        ba.containerwidget(edit=self._root_widget, transition="out_right",)
+
+# ba_meta export plugin
+
+
+class moodlight(ba.Plugin):
+    def __init__(self):
+        pass
+    Map._old_init = Map.__init__
+
+    def on_app_running(self):
+        try:
+            _ba.timer(0.5, self.on_chat_message, True)
+        except Exception as err:
+            Print(err)
+
+    def on_chat_message(self):
+        messages = _ba.get_chat_messages()
+        if len(messages) > 0:
+            lastmessage = messages[-1].split(":")[-1].strip().lower()
+            if lastmessage in ("/mood light", "/mood lighting", "/mood_light", "/mood_lighting", "/moodlight", "ml"):
+
+                with open("moodlightSettings.txt", "r") as mltxt:
+                    global Ldefault, Udefault
+                    data = mltxt.read()
+                    Ldefault, Udefault = data.split("\n")
+                    Ldefault = int(Ldefault)
+                    Udefault = int(Udefault)
+                SettingWindow()
+                _ba.chatmessage("Mood light settings opened")
+
+    def on_plugin_manager_prompt(self):
+        SettingWindow()
+
+    def _new_init(self, vr_overlay_offset: Optional[Sequence[float]] = None) -> None:
+        self._old_init(vr_overlay_offset)
+        in_game = not isinstance(_ba.get_foreground_host_session(), mainmenu.MainMenuSession)
```

Comment:
i think this won't handle cases say when you are on the score screen after a round ends, not a blocker and can be addressed in future versions tho